chaos-test/chaostest/chaostest/aws/awsutils.py

```diff
@@ -0,0 +1,131 @@
+"""Utility class for aws init and gathering operations in chaos"""
+
+import time
+import logging
+import random
+
+import boto3
+import requests
+from boto3 import Session
+from chaostest.kubernetes.k8sutils import K8sUtils
+from chaostest.utils.chaos_custom_exception import ChaosTestException
+
+logger = logging.getLogger(__name__)
+
+__author__ = 'Vijay Thomas'
+
+INVALID_RESOURCE = "Not supported Resource"
+IAM_VALIDATE_URL = "http://169.254.169.254/latest/meta-data/iam/security-credentials/"
```

Comment:
We should avoid giving .. ip ..

Reply:
Actually its from here : https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/iam-roles-for-amazon-ec2.html#launch-instance-with-role. 169.254.169.254 is the address of the AWS metadata service,,

Reply:
cool